apache/superset
# feat: Create/onsave Email Report functionality for Dashboard and Chart

### SUMMARY
Implemented the Create/onsave Email Report functionality for Dashboard and Chart.

Bonus: Modal styling now matches more closely to Figma.

### SCREENSHOT
<img width="453" alt="Screen Shot 2021-07-23 at 12 27 38 AM" src="https://user-images.githubusercontent.com/55605634/126742505-da25082e-6b36-4681-a5a0-5aa428ebecb2.png">

### TESTING INSTRUCTIONS
- Click "Dashboards" in the upper navigation bar
- Navigate to any Dashboard
- Click the Calendar icon button
	- Located at the top of the chart between the "Edit dashboard" and the "..." button
- Observe the email report modal

### ADDITIONAL INFORMATION
